contents.css

```diff
@@ -133,3 +133,47 @@ a > img {
 	border: none;
 	outline: 1px solid #0782C1;
 }
+
+.wcm-slider__images {
+	margin-bottom: 1rem;

```

Comment:
Deze styling moet in de module en niet in de core van het wcm. Kan je deze verwijderen?
Ik heb ze al overgenomen in de module.